modules/aggregation-layers/src/contour-layer/contour-layer.ts

```diff
@@ -286,8 +286,8 @@ export default class GridLayer<DataT = any, ExtraPropsT extends {} = {}> extends
 
   override draw(opts) {
     // Replaces render time viewport with our own
-    if (opts.moduleParameters.viewport) {
-      opts.moduleParameters.viewport = this.state.aggregatorViewport;
+    if (opts.moduleParameters.project) {
```

Comment:
- Maybe `moduleProps` is more appropriate? 
- `parameters` usually refer to GPU parameters (GLParameters equivalent)